public/css/feature-tip-group.css

```diff
@@ -111,6 +111,10 @@
     max-width: 600px;
   }
 
+  .feature-tip-headline[data-lang="de"] {
```

Comment:
This all seems very locale specific. Do we know it doesn't happen in other locales? Or is German just the most verbosest thing and the only thing breaking?

Reply:
Refactored my Puppeteer scraper and created a new repo at https://github.com/pdehaan/blurts-mobile-screenshots

Looks like German (de) is the only locale [with at least 80% translations in Pontoon] where that specific text wraps on an iPhone viewport in portrait mode. In landscape mode it didn't seem to have the issue.

I have all the screenshots in the repo if you're curious. Ignore the weird bug where setting the viewport to 4000px high just makes the header super big and not the whole page. Not sure what's going on there.